src/components/custom-aggrid/custom-aggrid-header.js

```diff
@@ -247,6 +261,21 @@ const CustomHeaderComponent = ({
                                         </IconButton>
                                     </Grid>
                                 )}
+                                {isColumnSecSorted && (
+                                    <Grid item>
+                                        <IconButton>
+                                            {secSortWay === SORT_WAY.asc ? (
+                                                <ArrowUpward
+                                                    sx={styles.iconSize}
+                                                />
+                                            ) : (
+                                                <ArrowDownward
+                                                    sx={styles.iconSize}
+                                                />
+                                            )}
+                                        </IconButton>
+                                    </Grid>
```

Comment:
check with designer if it's enough to display "secondary filter"
I would display a "2" next to the children sort arrow

Reply:
I did last week. He didn't like the "2" idea, and made a proposition which looks better but won't be done for now. I precised it in the main US.